src/services/projectService.js

```diff
@@ -1,53 +1,83 @@
-import {app} from "../config/config";
+import { app } from "../config/config";
 import ENDPOINTS from "../constants/endpoints";
 import HEADERS from "../constants/headers";
-import {TOKEN} from "../constants/components";
+import { TOKEN } from "../constants/components";
 import STATUS_CODES from "../constants/statusCodes";
-import {postRequest} from "../clients/restClient";
-import {log} from "nodemon/lib/utils";
+import { postRequest } from "../clients/restClient";
+import { log } from "nodemon/lib/utils";
+import {
+  getCompanyName,
+  getProductDescription,
+  getRandomBoolean,
+} from "../utils/faker";
+import { getAnnouncementProject, getNameProject } from "../pages/projectPageUi";
+import { expect } from "chai";
 
 const DEFAULT_PROJECT = {
-    name: "New project using API",
-    announcement: "This is the description for the project",
-    show_announcement: true,
-}
+  name: "New project using API",
+  announcement: "This is the description for the project",
+  show_announcement: true,
+};
+
+export const project = {
+  name: getCompanyName(),
+  announcement: getProductDescription(),
+  show_announcement: getRandomBoolean(),
+};
```

Comment:
Ok
If we deside to create another project will define it here again like `project2`??